src/assets/themes/colors.jsx

```diff
@@ -0,0 +1,24 @@
+const yellow = '#FFBC33';
+const black = '#000';
+const white = '#FFF';
+const red = '#EB5757';
+const shadow = 'rgba(0, 0, 0, 0.1)';
+const grey = '#E3E2E7';
+const grey1 = '#F6F7FB';
+const grey_text = '#858598';
+const grey_text1 = '#A6ABB9';
+const grey_text2 = '#FAFAFA';
+
+export const YELLOW = yellow;
```

Comment:
I think we should avoid such color names as YELLOW, GREEN, and so on. The main reason to save colors as variables is to make redesign or recoloring easier.
FE designer changed `yellow` to `orange`. You will open this file and change Y`ELLOW = orange` which doesnt make sense. 
Instead, you could name it ACCENT or MAIN or PRIMARY or SECONDARY if its one of general colors or name it more specific if it uses for some specific places like LOGIN_BTN or BUTTON_BACKGROUND or etc